teachopencadd/talktorials/019_One_Hot_Encoding/talktorial.ipynb

```diff
@@ -0,0 +1,2926 @@
+{
```

Comment:
<p>the code shoule be:</p><pre class="ql-syntax" spellcheck="false">df = pd.read_csv(DATA/'CHEMBL25_activities_EGFR.csv',
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;lineterminator='\n', index_col=0)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Done editing :) But I would like to ask what difference does it make ?</p>

Reply:
<p>It's different betwee </p><p><code>'DATA/CHEMBL25_activities_EGFR.csv'</code></p><p>and</p><p><code>DATA/'CHEMBL25_activities_EGFR.csv'</code></p><p> </p>

Reply:
<p>Also, I suggest use <code>"</code>  instead of <code>'</code>  for string in the whole notebook.</p>

Reply:
Use `black-nb -l 99 <notebook.ipynb>` to autoformat the talktorials, no need to do it manually!